register.py

```diff
@@ -0,0 +1,101 @@
+""" Register user with pre-commit usage reporting """
+from __future__ import print_function
+
+import json
+import sys
+
+from runner import run
+
+try:
+    from past.builtins import raw_input  # type: ignore
+except ImportError:
+    pass
```

Comment:
@danjoneslf I've fixed it in the way I think is best. This'll fail python 2 syntax check, but - critically - pass our dev flake8!